.bumpversion.cfg

```diff
@@ -3,5 +3,3 @@ current_version = 0.10.1
 commit = True
 message = [skip ci] Bump version: {current_version} → {new_version}
 tag = False
-
-[bumpversion:file:src/con_duct/__main__.py]
```

Comment:
This whole file should be deleted if you're switching to `versioningit`, and the "afterChangelog" line in `.autorc` should be deleted.